adapter/src/dbt/adapters/fal_experimental/utils/environments.py

```diff
@@ -179,16 +250,29 @@ def _get_dbt_packages(is_teleport: bool = False) -> Iterator[Tuple[str, Optional
 
     dbt_fal_dep = "dbt-fal"
     if _is_pre_release(dbt_fal_version):
-        dbt_fal_path = _get_adapter_root_path()
-        if dbt_fal_path is not None:
-            # Can be a pre-release from PyPI
-            dbt_fal_dep = str(dbt_fal_path)
+        if is_remote:
+            # If it's a pre-release and it's remote, its likely us developing, so we try installing
+            # from Github and we can get the custom branch name from FAL_GITHUB_BRANCH environment variable
+            # TODO: Handle pre-release on PyPI. How should we approach that?
```

Comment:
I think we may not need to handle this because we want to remove the `_is_pre_release` check for something that assures us we are in dev mode and not just a good guess like that function call. Once we have that, this problem goes away.